### DISCUSSION TITLE
launching into google colab with a pre-built environment.

### DISCUSSION THREAD

**Discussion by sam1mitchell1-hub** [executablebooks/meta#927] [Q&A]
Hi there, is it possible to launch from a jupyterbook into google colab with an environment that is specified. It is possible to do this using binder, where binder launches into jupyterhub rather than google colab. I have been finding that this way of doing it is inconsistent and doesnt always launch properly so I was wondering if it is possible to somehow create the environment in google colab, without having to do it manually, for example using some kind of postBuild file like with binder.

**Comment by choldgraf** [comment]
I don't believe that this is possible in colab, though you could include a bunch of `%pip install` statements at the top of your notebook or something. Maybe that would work?

If others in the community have experience with colab please share your thoughts!

**Reply by sam1mitchell1-hub** [reply]
Thank you for the response. This was my worry too. 

No worries, I will find a work around.
